templates/newrelic-stack-set.yml

```diff
@@ -32,14 +36,14 @@ Resources:
     Condition: UseDefaultPolicy
     Properties:
       ManagedPolicyArns: 
-        - arn:aws:iam::aws:policy/ReadOnlyAccess
+        - !Sub 'arn:${AWS::Partition}:iam::aws:policy/ReadOnlyAccess'
       RoleName: !Join ['_', ['NewRelicIntegrationRole', !Ref NewRelicAccountNumber]]
       AssumeRolePolicyDocument:
         Version: 2012-10-17
         Statement:
           - Effect: Allow
             Principal:
-              AWS: 'arn:aws:iam::754728514883:root'
+              AWS: !Sub 'arn:aws:iam::${NewRelicTrustedPrincipal}:root'
```

Comment:
Does this needs to be 'root'? or can it be a different user?

Reply:
since NewRelic AWS account may use different role to perform STS, we need to assign it to root to allow IAM users/roles with proper external ID to assume it.